source/mbed_platform.cpp

```diff
@@ -61,8 +66,22 @@ uint32_t getTimeOverflows(){
 }
 
 void sleepFromUntil(tick_t now, tick_t until){
+
+#if YOTTA_CFG_MINAR_TEST_CLOCK_OVERFLOW
+    tick_t timer_top_bits = lp_ticker_read() & ~Time_Mask;
```

Comment:
shall we add a comment to this code here? I cant find a short description in the commit message or in this code section - Why now, until and real_now are updated with the top_bits.